cms/static/js/views/settings/grading.js

```diff
@@ -96,7 +102,13 @@ var GradingView = ValidatingView.extend({
         };
         var grace_period = this.model.get('grace_period');
         this.$el.find('#course-grading-graceperiod').val(
-            format(grace_period.hours) + ':' + format(grace_period.minutes)
+            StringUtils.interpolate(
```

Comment:
You can leave this, but this isn't necessary.  The `interpolate` function is really helpful with strings that require translation.  Since this entire string is just the time, you could just use the original string.